backend/utils.js

```diff
@@ -96,6 +97,10 @@ function Utils() {
 		} , {});
 	};
 
+	this.getRandomBytesToHexString = (length = 16) => {
```

Comment:
2 things:

1. Can we rename this to `generateHashString`? as this is esssentially the purpose of this function
2. It's confusing for the caller to pass in a length but get a length *2 string length as a return. Can we make it so it takes in 2x length so it matches what the string length would be ultimately? i.e
```suggestion
	this.getRandomBytesToHexString = (length = 32) => {
	    return crypto.randomBytes(length/2).toString("hex");
```